services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -230,6 +230,40 @@ public void build_walkingAlleyBiasOptions() {
     assertTrue(directions.cloneCall().request().url().toString().contains("alley_bias=1.0"));
   }
 
+  @Test
+  public void build_arrivedByOption(){
+    LocalDateTime arriveBy = LocalDateTime.of(2020, Month.SEPTEMBER, 22, 9, 47);
+
+    MapboxDirections directions = MapboxDirections.builder()
+      .destination(Point.fromLngLat(13.4930, 9.958))
+      .origin(Point.fromLngLat(1.234, 2.345))
+      .accessToken(ACCESS_TOKEN)
+      .arriveBy(arriveBy)
+      .build();
+
+    assertEquals(
+            directions.cloneCall().request().url().queryParameter("arrive_by"),
+            FormatUtils.formatDateTimeToIso8601(arriveBy)
```

Comment:
I guess we also need a test that `FormatUtils.formatDateTimeToIso8601` returns a String with needed pattern 
to check  if we get a valid `iso8601` String 
something like this:
assertEquals("2020-11-12'T'22:22", FormatUtils.formatDateTimeToIso8601(input))
wdyt?

Reply:
agree, but it has to be tests of FormatUtils, not MapboxDirections. Will add them 👍